src/WebJobs.Script.WebHost/Diagnostics/FunctionInstanceLogger.cs

```diff
@@ -43,7 +48,9 @@ internal class FunctionInstanceLogger : IAsyncCollector<FunctionInstanceLogEntry
                 var tableProvider = LogFactory.NewLogTableProvider(client);
 
                 string containerName = Environment.MachineName;
-                this._writer = LogFactory.NewWriter(hostName, containerName, tableProvider, (e) => OnException(e, trace));
+                _hostId = hostId;
```

Comment:
Wondering why the HostID needs to be logged. We're already logging the AppName column, and in Functions we default the HostID to the AppName. How would this be used - do we actually need it?